schema.py

```diff
@@ -0,0 +1,43 @@
+import graphene
+
+from graphene import relay, Schema
+from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
+from models import User, Story, db_session
+
+
+class UserType(SQLAlchemyObjectType):
+    class Meta:
+        model = User
+        interfaces = (relay.Node, )
+
+
+class StoryType(SQLAlchemyObjectType):
+    class Meta:
+        model = Story
+        interfaces = (relay.Node, )
+
+
+class CreateStory(graphene.Mutation):
+    class Arguments:
+        story = graphene.String()
+        user_id = graphene.Int()
+    story = graphene.Field(StoryType)
+
+    def mutate(self, info, story, user_id):
+        story = Story(story=story, user_id=user_id)
+        db_session.add(story)
```

Comment:
Consider using the good practice of fat models, supply methods in the model to save new objects, this will prevent you from using session.add and session.commit in every place where saving is involved, as such we shall avoid repetition

Reply:
Noted, but I have never used the concept of fat models, so I am going to first research about what fat models are and see how to apply them as requested.